content/assignments/techjobs-console/_index.md

```diff
@@ -0,0 +1,477 @@
+---
+
+## Introduction
+
+Congratulations! Based on your hard work and strong coding skills, you have
+been brought on as an apprentice to the LaunchCode Tech Team. You will be
+paired with a mentor to help you get comfortable and continue learning.
+
+The Company Team at LaunchCode works with employer partners to match qualified
+programmers with apprenticeships. They asked for a new tool to be built to
+help them easily manage data for currently available jobs. Over the next few
+weeks, you will help them build this application alongside mentors from the
+Tech Team.
+
+This first project will be a simple proof-of-concept prototype. It won't be
+pretty or have lots of features, but it will give you a chance to work through
+some initial concepts and get feedback from LaunchCode staff.
+
+Your mentor on this project is Blake.
+
+{{< rawhtml >}}
+   <img src="pictures/LC-Blake.png" alt="LaunchCode Mentor Blake" width=20% />
+{{< /rawhtml >}}
+
+### Learning Objectives
+
+In this project, you will show that you can:
+
+1. Read and understand code written by others.
+1. Use core C# syntax (methods, variables, loops, conditionals).
+1. Utilize `List` and `Dictionary` collection types.
+1. Work with console I/O via the `Console` class.
+1. Work with data types and arrays.
+
+## TechJobs (Console Edition)
+
+The app you must help design is a simple console (i.e. command-line) prototype
+of the new TechJobs app. It will allow LaunchCode staff to browse and search
+listings of open jobs by employer partners.
+
+The prototype process gives everybody a chance to work out some initial ideas
+without investing a ton of time into developing a finished product. Once
+everybody likes the prototype, the Tech Team will begin work toward a
+full-fledged application.
+
+### Your Assignment
+
+Blake created a console application and started to fill in some features. His
+code allows users to search job listings by one of several fields. It can also
+display lists of all of the values of a given field in the system (e.g. all
+employers, or all locations).
+
+Blake has now handed the task off to you. You must add a couple of features and
+then get feedback from the Company Team.
+
+After you work through the tasks Blake has laid out for you, tackle one
+or more of the [bonus missions]({{< relref "./#bonus-missions" >}}).
+
+## Getting Started
+
+<!-- TODO: Add link back to Assignment 0 -->
+
+In Canvas, **Graded Assignment #1: TechJobs Console** contains a GitHub Classroom assignment invitation link and then set up the project in Visual Studio. Refer back to the GitHub Classroom instructions from Assignment 0 for details. 
+
+Before diving in and starting to code, make sure you understand what the code
+you've been given does. Since you're starting with a functioning---albeit
+unfinished---program, go ahead and run it to get an idea of how it works. To do
+this, right-click on the project that contains the `TechJobs` class and select *Run Project*.
```

Comment:
This is a bit confusing - I right clicked on TechJobsConsoleAutograded6

Reply:
Changed the language a bit to help address confusion and encourage students to click around and explore.